dapp/src/components/CollapsibleInfo.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Accordion, AccordionContent, AccordionHeader, Title } from '@iota/apps-ui-kit';
+import { useState } from 'react';
+
+type AccordionInfoProps = {
+    title: string;
+    children: React.ReactNode;
+};
+
+export function CollapsibleInfo({ title, children }: AccordionInfoProps) {
```

Comment:
i'd rename this to just Collapsible
```suggestion
export function Collapsible({ title, children }: AccordionInfoProps) {
```